diagnostic.md

```diff
@@ -32,6 +40,11 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+Getters:
+.find;
+.html
+Setters:
+.addClass
```

Comment:
Please read these articles on get: http://www.w3schools.com/jquery/jquery_dom_get.asp
and set: http://www.w3schools.com/jquery/jquery_dom_set.asp